Samples/Hast.Samples.Consumer/Gui.cs

```diff
@@ -0,0 +1,356 @@
+using Hast.Layer;
+using Hast.Samples.Consumer.Models;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Terminal.Gui;

```

Comment:
Fog ez működni a Zynqen is? Próbáld ki kérlek.

Reply:
Igen működik. Annyi, hogy SSH-ban nem működik az egér, de billentyűzettel teljesen jól kezelhető. Az egész folyamaton végigmentem. 

A docker image-ben mentettem egy setupot Generate Hardware Only-val és azzal buildeltem.
![image](https://user-images.githubusercontent.com/4007293/140625655-bfb4e5dd-8574-4a22-b928-ed7b993ab13a.png)

Utána a kapott fájlokat meg egy új app publisht feltöltöttem SFTP-n és onnan szintén le tudtam futtatni:
![image](https://user-images.githubusercontent.com/4007293/140625643-50d6744a-f894-40cb-af7c-cb395de16645.png)

Reply:
Viszont csinálnom kellett hozzá egy külön branchot ahol a HAST-174-el kapcsolatos változtatások vannak.
- https://github.com/Lombiq/Hastlayer-Core/pull/9
- https://github.com/Lombiq/Hastlayer-SDK/pull/60

Reply:
De az egész megy scriptelve is, nem? Szal mondjuk ilyen `run-demo.sh`-szerű megoldás (tudom, az a Demo project), az egész egy felparaméterezett CLI commanddal végigfuttatható? Mert ilyen embedded környezetben inkább az lenne, a GUI akkor érdekes, ha amúgy tényleg grafikus környezetben csinálod.

Reply:
Egy scriptbe a parancssori paramétereket használnád ugyanúgy mint eddig. Azok továbbra is működnek (lásd `FromCommandLine`). Illetve a mentett configot a `-load configname` paranccsal is le lehet hívni közvetlenül, ami átjárást biztosít a parancssor és a gui között

Reply:
Akkor a zynq-benchmark scriptnek nem kell változnia?

Reply:
Nem kell. Backwards compatible.

Reply:
Cool.